run_uh.py

```diff
@@ -67,10 +67,9 @@ def exit_with_error(title, message):
 
 def check_python_version():
 	# python up to version 2.6.1 returns an int. http://bugs.python.org/issue5561
-	if platform.python_version_tuple()[0] not in (2,'2'):
-		exit_with_error('Unsupported Python version', 'Python 2 is required to run Unknown Horizons.')
-
-
+    if float(sys.version[:3]) < 2.7:
```

Comment:
That's certainly an interesting way to do it, but I don't think using `float` here is a good idea. Why not do this instead:

`if sys.version[:3] < (2, 7):`

The comment above can then be deleted, since it refers to `python_version_tuple` specifically.